hasura/metadata/databases/ibcjuno/tables/public_token_unit.yaml

```
@@ -0,0 +1,43 @@
+table:
+  name: token_unit
+  schema: public
+object_relationships:
+- name: token
+  using:
+    foreign_key_constraint_on: token_name
+- name: token_price
```

I think we can change this to be simply `price` for easier usage 
```suggestion
- name: price
```